nestjs/src/courses/interviews/interviews.service.ts

```diff
@@ -23,7 +23,7 @@ export class InterviewsService {
 
   public getAll(courseId: number) {
     return this.courseTaskRepository.find({
-      where: { courseId, type: 'interview' },
+      where: { courseId, type: In(['interview', 'stage-interview']), disabled: false },
```

Comment:
@apalchys could u please have a look. I think shouldn't break smth, but just for another pair of eyes.